### ISSUE TITLE
CRDs of Gemini not installed properly

### ISSUE THREAD

**Issue by jelel-fliss** [FairwindsOps/gemini#48]
`unable to recognize "gemini-backup.yaml": no matches for kind "SnapshotGroup" in version "gemini.fairwinds.com/v1beta1"`

Despite following the proper instructions of the installation of gemini in my Kubernetes cluster, I still can't create snapshot groups to backup volumes. This was also posted on Stackoverflow, but no answer was given. 

https://stackoverflow.com/questions/65732647/how-to-backup-pvc-regularly

**Comment by sudermanjr** [comment]
@jelel-fliss it looks like we need to update that chart with the helm 3 CRDs pattern to make it work. 

If you install the CRD manually, does it work for you?

**Comment by sudermanjr** [comment]
Okay, so I sit corrected. We are apparently installing the CRD from the codebase. My guess is that the apiextensionsv1beta1 isn't working because it has been removed in your k8s version. Can you verify which version you're using?

https://github.com/FairwindsOps/gemini/blob/main/pkg/types/snapshotgroup/v1beta1/crd.go

**Comment by jelel-fliss** [comment]
> Okay, so I sit corrected. We are apparently installing the CRD from the codebase. My guess is that the apiextensionsv1beta1 isn't working because it has been removed in your k8s version. Can you verify which version you're using?
> 
> https://github.com/FairwindsOps/gemini/blob/main/pkg/types/snapshotgroup/v1beta1/crd.go

Hello @sudermanjr , thank you for paying attention to the issue. Here is the version of Kubernetes that I am using

```
ubuntu@kubemaster:~$ kubectl version
Client Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.1", GitCommit:"632ed300f2c34f6d6d15ca4cef3d3c7073412212", GitTreeState:"clean", BuildDate:"2021-08-19T15:45:37Z", GoVersion:"go1.16.7", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.3", GitCommit:"c92036820499fedefec0f847e2054d824aea6cd1", GitTreeState:"clean", BuildDate:"2021-10-27T18:35:25Z", GoVersion:"go1.16.9", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by sudermanjr** [comment]
Yep, that's exactly the issue. We need to update the code to install the CRD at the newer version.

**Comment by nooshin-mirzadeh** [comment]
Any plan for fixing this issue?

**Comment by sudermanjr** [comment]
@rbren  Is going to take a look and see how hard this will be to tackle. Thanks all!
